src/securityinsight/azext_sentinel/manual/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
```

Comment:
this module has no manual code?

Reply:
> this module has no manual code?

@yungezz Actually it has. I will split the manual code from the mixed generated code.